oct_levitation:
  # rigid_body: "onyx_disc_80x15" # Use name registered in rigid_bodies.REGISTERED_BODIES
  # rigid_body: "onyx_disc_80x15_I40"
  rigid_body: "onyx_disc_80x15_I40_N52"
  # rigid_body: "onyx_disc_80x7_5"